src/config/routes.js

```diff
@@ -4,6 +4,7 @@ import LoginPage from '../components/login/login.container';
 import TrailMap from '../components/trail-map/trail-map.container';
 import ReportProblem from '../components/report-problem/report-problem.container';
 import ReportProblemIssueType from '../components/report-problem-issue-type/report-problem-issue-type.container';
+import ReportProblemComment from '../components/report-problem-comment/report-problem-comment.container';
```

Comment:
I don't understand how this works.  We might need to chat on Skype.  When I remove the imports from the route.js, the linter doesn't like it and the app doesn't complete.  So I guess it somehow knows index.js as acting as an interface and will auto import?  Well, once you explain it to me then update it.